docs/docs/integrate/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Integrate Rill into your Application
+sidebar_label: Integrate
+sidebar_position: 0
+---
+
+Extend Rill's capabilities by integrating dashboards and data into your existing applications and workflows.
+
+## Embed Rill into your Application
+
+- **[Embed Dashboards](/integrate/embedding)** - Embed Rill dashboards directly into your applications using iframes
+- **[Control your Embedded Dashboards](/integrate/embed-iframe-api)** - Programmatically control embedded dashboards from your application
+- **[See it Live](https://rill-embedding-example.netlify.app/)** - View a live example of embedded Rill dashboards
+
+## Custom APIs
+- **[Build Custom APIs](/integrate/custom-apis)** - See the different types of APIs to build
+- **[Integrate with Custom APIs](/integrate/custom-api)** - Build custom APIs to retrieve aggregated data from your metrics views
```

Comment:
Nit: These descriptions don't seem to cleanly match the "Build Custom APIs" and "Integrate with Custom APIs" titles